decimal.go

```diff
@@ -925,13 +939,13 @@ func (d *Decimal) UnmarshalJSON(decimalBytes []byte) error {
 
 	str, err := unquoteIfQuoted(decimalBytes)
 	if err != nil {
-		return fmt.Errorf("Error decoding string '%s': %s", decimalBytes, err)
+		return err
```

Comment:
The error returned by `unquoteIfQuoted` is still made by `fmt.Errorf`.

Reply:
It is now ErrorInvalidType?